weave

```diff
@@ -28,6 +28,8 @@ usage() {
     echo "weave start      <cidr> [<cidr> ...] <container_id>"
     echo "weave attach     <cidr> [<cidr> ...] <container_id>"
     echo "weave detach     <cidr> [<cidr> ...] <container_id>"
+    echo "weave add-dns    <cidr> [<cidr> ...] <container_id> [-fqdn fqdn]"
```

Comment:
There is an inconsistency in our option naming here. `weave expose` takes a `-h <hostname>` parameter, which serves the same purpose. I'm leaning towards changing that to `-fqdn <fqdn>` - seems like a more accurate description of what it is. But I'd like to hear from @awh on this.

Also note that user-supplied input should appear in angle brackets in the usage. So it should be `-fqdn <fqdn>`.